src/App.tsx

```diff
@@ -1,9 +1,15 @@
+import { About } from './components/About/About';
+import { Hero } from './components/Hero/Hero';
+import { Navbar } from './components/Navbar/Navbar';
+import styles from './styles/App.module.css'
+
 
 export function App() {
   return (
-    <>
-      <h1>Portfolio</h1>
-    </>
-  )
-}
-
+    <div className={styles.App}>
+      <Navbar />
+      <Hero />
+     <About />
+    </div>
+  );
```

Comment:
Can you fix the indentation here ?
```suggestion
    <div className={styles.App}>
      <Navbar />
      <Hero />
      <About />
    </div>
  );
```

Reply:
fixed https://github.com/Sakshi-Simform/portfolio/pull/2/commits/806e9c821988805a9ab44516d8b41f942e61cdfb